rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/190_percentiles_hdr_metric.yml

```diff
@@ -0,0 +1,447 @@
+              meta:
+                foo: bar
+              percentiles:
+                field: int_field
+                hdr: {}
+
+  - match: { hits.total: 4 }
+  - length: { hits.hits: 4 }
+  - match: { aggregations.percentiles_int.meta.foo: "bar" }
+  - match:
+      aggregations.percentiles_int.values:
+        1.0: 1.0
+        5.0: 1.0
+        25.0: 1.0
+        50.0: 51.0302734375
+        75.0: 101.0615234375
+        95.0: 151.1240234375
+        99.0: 151.1240234375
+
+---
+"Invalid params test":
+
+  - do:
+      catch: /\[numberOfSignificantValueDigits\] must be between 0 and 5/
+      search:
+        body:
+          aggs:
+            percentiles_int:
+              percentiles:
+                field: int_field
+                hdr:
+                  number_of_significant_value_digits: -1
+
+  - do:
+      catch: /\[numberOfSignificantValueDigits\] must be between 0 and 5/
+      search:
+        body:
+          aggs:
+            percentiles_int:
+              percentiles:
+                field: int_field
+                hdr:
+                  number_of_significant_value_digits: 10
+
+  - do:
+      catch: request
+      search:
+        body:
+          aggs:
+            percentiles_int:
+              percentiles:
+                field: int_field
+                hdr:
+                  number_of_significant_value_digits: null
+
+  - do:
+      catch: /\[percents\] must not be empty/
+      search:
+        body:
+          aggs:
+            percentiles_int:
+              percentiles:
+                field: int_field
+                hdr: {}
+                percents: []
+
+  - do:
+      catch: request
+      search:
+        body:
+          aggs:
+            percentiles_int:
+              percentiles:
+                field: int_field
+                hdr: {}
+                percents: null
+
+  - do:
+      catch: request
+      search:
+        body:
+          aggs:
+            percentiles_int:
+              percentiles:
+                field: int_field
+                hdr: {}
+                percents: ["foo"]
+
+  - do:
+      catch: request
+      search:
+        body:
+          aggs:
+            percentiles_string:
+              percentiles:
+                field: string_field
+                hdr: {}
+
+---
+"Explicit Percents test":
+
+  - do:
+      search:
+        body:
+          aggs:
+            percentiles_int:
+              percentiles:
+                field: int_field
+                percents: [5.0, 25.0, 50.0]
+                hdr: {}
+            percentiles_double:
+              percentiles:
+                field: double_field
+                percents: [5.0, 25.0, 50.0]
+                hdr: {}
+
+
+  - match: { hits.total: 4 }
+  - length: { hits.hits: 4 }
+  - match:
+      aggregations.percentiles_int.values:
+        5.0: 1.0
+        25.0: 1.0
+        50.0: 51.0302734375
+  - match:
+      aggregations.percentiles_double.values:
+        5.0: 1.0
```

Comment:
I tink these values need to be quoted

```"5.0": 1.0```

Cause this is failing in the python client.

Reply:
That is consistent with the documentation for all versions I checked, from v5.3 though master, as well as 2.4 and 1.7. Seems like the test is wrong.